lockfile.json

```diff
@@ -111,7 +111,7 @@
     "revision": "a22fa5e19bae50098e2252ea96cba3aba43f4c58"
   },
   "teal": {
-    "revision": "6142154f8d78dbf00b9c0e70bf2f994d97e76582"
+    "revision": "cec82d45e630005346d8e9df2414afedd212362f"
```

Comment:
We don't want this teal update here.
We'll let CI update that one.

Reply:
That is the commit of the CI

Reply:
Sorry, did not saw that !